spec/ndx-extracellular-channels.extensions.yaml

```diff
@@ -0,0 +1,349 @@
+      brain area, e.g., "histology", "MRI".
+    required: false
+  datasets:
+  - name: contact
+    neurodata_type_inc: DynamicTableRegion
+    doc: The row in a ContactsTable that represents the contact used as a channel.
+  - name: reference_contact
+    neurodata_type_inc: DynamicTableRegion
+    doc: The row in a ContactsTable that represents the contact used as a reference.
+    quantity: '?'
+  - name: filter
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: The filter used on the raw (wideband) voltage data from this contact, including
+      the filter name and frequency cutoffs, e.g., "High-pass filter at 300 Hz."
+    quantity: '?'
+  - name: estimated_position_ap_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Anteroposterior (AP) stereotactic coordinate of the estimated contact position,
+      in millimeters. + is anterior. Coordinate is relative to the zero-point described
+      in `position_reference`.
+    quantity: '?'
+  - name: estimated_position_ml_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Mediolateral (ML) stereotactic coordinate of the estimated contact position,
+      in millimeters. + is right. Coordinate is relative to the zero-point described
+      in `position_reference`.
+    quantity: '?'
+  - name: estimated_position_dv_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Dorsoventral (DV) stereotactic coordinate of the estimated contact position,
+      in millimeters. + is up. Coordinate is relative to the zero-point described
+      in `position_reference`.
+    quantity: '?'
+  - name: estimated_brain_area
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: The brain area of the estimated contact position, e.g., "CA1".
+    quantity: '?'
+  - name: confirmed_position_ap_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Anteroposterior (AP) stereotactic coordinate of the confirmed contact position,
+      in millimeters. + is anterior. Coordinate is relative to the zero-point described
+      in `position_reference`.
+    quantity: '?'
+  - name: confirmed_position_ml_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Mediolateral (ML) stereotactic coordinate of the confirmed contact position,
+      in millimeters. + is right. Coordinate is relative to the zero-point described
+      in `position_reference`.
+    quantity: '?'
+  - name: confirmed_position_dv_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Dorsoventral (DV) stereotactic coordinate of the confirmed contact position,
+      in millimeters. + is up. Coordinate is relative to the zero-point described
+      in `position_reference`.
+    quantity: '?'
+  - name: confirmed_brain_area
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: The brain area of the actual contact position, e.g., "CA1".
+    quantity: '?'
+  groups:
+  - name: probe_insertion
```

Comment:
Can you guys @rly@ CodyCBakerPhD remind me why we decided to include the insertion in the channels table and not in the Probe? The probes are the things that are inserted so I think we had a good reason but I can't really recall. Do you?

Reply:
I think the only reason might have been that because it contains position information, it is useful attached to the channels table where there is also position information. But I see your point that it is more a property of the `Probe`. Happy to make that change. (and tagging @CodyCBakerPhD correctly)

Reply:
Yes, I would prefer that. For the position information I am leaning to delegate that to your proposal in #3 of a `AnatomicalCoordinatesTable`.